### PR TITLE
feat: customization service append and customize functionality should run once

### PR DESCRIPTION
**Pull request by wayfarer3130**
#### PR

- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] OS: 
- [] Node version: 
- [] Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: extensions/cornerstone-dynamic-volume/src/panels/WorkflowPanel.tsx
Comment (wayfarer3130) [inline extensions/cornerstone-dynamic-volume/src/panels/WorkflowPanel.tsx:1]: Fixing the use of the customization service to be correct.

File: extensions/cornerstone/jest.config.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ const pkg = require('./package');
 
 module.exports = {
   ...base,
-  name: pkg.name,
```
Comment (wayfarer3130) [inline extensions/cornerstone/jest.config.js:6]: name isn't a valid field, was causing spurious warnings during the unit test phase of this change.

File: platform/core/src/services/CustomizationService/CustomizationService.test.js [hunk 2]
```diff
@@ -160,5 +165,149 @@ describe('CustomizationService.ts', () => {
       expect(result.label).toBe(testItem.label);
       expect(result.value).toBe(props.testAttribute);
     });
+
```
Comment (wayfarer3130) [inline platform/core/src/services/CustomizationService/CustomizationService.test.js:168]: Add a whole bunch of unit tests to show how the services can be used/how to handle the append etc.

File: platform/core/src/services/CustomizationService/CustomizationService.ts
Comment (wayfarer3130) [inline platform/core/src/services/CustomizationService/CustomizationService.ts:1]: Adds default customizations, which can be used to provide a basis for other changes, and added working merge and append functionality.

File: platform/docs/docs/platform/services/ui/customization-service.md
Comment (wayfarer3130) [inline platform/docs/docs/platform/services/ui/customization-service.md:1]: Adding documentation on how to use the new functionality.

File: platform/ui/package.json [hunk 3]
```diff
@@ -51,7 +51,7 @@
     "react-dnd": "14.0.2",
     "react-dnd-html5-backend": "14.0.0",
     "react-dom": "^18.3.1",
-    "react-draggable": "4.4.3",
+    "react-draggable": "^4.4.6",
```
Comment (wayfarer3130) [inline platform/ui/package.json:54]: This fixes a persistent warning about react 18 incompatibilities.

**Review by sedghi — APPROVED** [review]
Thanks, looks great, i would simplify the append/merge in the docs later, i guess we can talk about the simple case instead of the advanced case

**Pull request merged**